tests/x11/gnucash.pm

```diff
@@ -27,7 +27,8 @@ sub run {
     # Leave tutorial window
     wait_screen_change { send_key 'alt-f4' };
     # Leave tips windows for GNOME case
-    if (check_var('DESKTOP', 'gnome') || check_var('DESKTOP', 'xfce')) {
+    if (check_var('DESKTOP', 'gnome') || check_var('DESKTOP', 'xfce') || check_var('DESKTOP', 'lxde')) {
+        send_key 'alt-tab' if (check_var('DESKTOP', 'lxde'));
```

Comment:
no, wait. Why do we have to press 'alt-tab' but only on LXDE? Is there a window not in focus? But shouldn't it be in focus? Could you reproduce this manually? Did you crosscheck e.g. xfce or gnome in the current state? Could be a product bug just as well

Reply:
The focus is per default on the main window when closing the tutorial window.
Imho this makes sense because the tips window is nothing crucial and would have been clicked away for a long time in a real use case

Reply:
yeah, but how is this lxde-specific?

Reply:
On gnome, KDE and XFCE it does not appear:
https://openqa.opensuse.org/tests/674069#step/gnucash/45 (xfce)
https://openqa.opensuse.org/tests/674064#step/gnucash/47 (gnome)
https://openqa.opensuse.org/tests/674060#step/gnucash/47 (kde)

Reply:
then find out why